tests/filtering_test.cc

```diff
@@ -1153,3 +1153,27 @@ SEASTAR_TEST_CASE(test_filtering) {
 
     });
 }
+
+SEASTAR_TEST_CASE(test_filtering_paging_and_aggregation) {
+    return do_with_cql_env_thread([] (cql_test_env& e) {
+        e.execute_cql("CREATE TABLE fpa (id int primary key, v int)").get();
+        for (int i = 0; i < 1000; ++i) {
+            e.execute_cql(format("INSERT INTO fpa (id, v) VALUES ({}, {})", i, i % 2)).get();
```

Comment:
(In the future,) Please consider using [`cquery_nofail`](https://github.com/scylladb/scylla/blob/master/tests/cql_assertions.hh#L82), which accurately reports failure location.  This is especially valuable for loop bodies like this.

Reply:
Neat. I've already done that for my newest series and I'll switch to `cquery_nofail` in the future. Thanks!